services/actions/.env.example

```diff
@@ -28,6 +28,9 @@ AWS_CONTENT_BUCKET_ID=<AWS bucket id>
 AWS_ELASTIC_TRANSCODER_NOTIFICATIONS_TOPIC_ARN=<AWS SNS topic for Elastic Transcoder notifications>
 AWS_ELASTIC_TRANSCODER_SERVICE_ROLE_ARN=<AWS Elastic Transcoder service role>
 AWS_MEDIACONVERT_SERVICE_ROLE_ARN=<AWS MediaConvert service role>
+AWS_MEDIALIVE_INPUT_SECURITY_GROUP_ID=<AWS ID of the security group to be used for MediaLive RTMP Push inputs>
+AWS_MEDIALIVE_NOTIFICATIONS_TOPIC_ARN=<AWS ARN of the SNS topic for MediaLive notifications>
+AWS_MEDIALIVE_SERVICE_ROLE_ARN=<AWS IAM role to be passed to MediaLive>
```

Comment:
These three vars aren't used in the actions service - only playout - so can be deleted from both here and the actions service README.